### ISSUE TITLE
ERROR occurs when cplex.opt file not available after solving.

### ISSUE THREAD

**Issue by OFR-IIASA** [iiasa/message_ix#338]
As cplex.opt file is created on the fly, at the beginning of each solve() command, it is then deleted upon completion of the solve command(). An error occurs when the cplex.opt file is not available - i.e. when trying to delete/unlink the cplex.opt file. This can occur when solving two scenarios in parallel and results in one of the processes returning an error.

**Comment by khaeru** [comment]
To be clear, are you running message_ix etc. in *separate processes*, e.g. two terminals each with a Python process importing and using message_ix?

**Comment by OFR-IIASA** [comment]
Yes, two separate processes are run.

**Comment by OFR-IIASA** [comment]
To clarify, the issue occurs after the solving process. i.e. when the file is to be deleted or in the code it is unlinked. Possibly unlink(missing_ok=True) schould resolve the issue.

```

Path.unlink(missing_ok=False)

    Remove this file or symbolic link. If the path points to a directory, use Path.rmdir() instead.

    If missing_ok is false (the default), FileNotFoundError is raised if the path does not exist.

    If missing_ok is true, FileNotFoundError exceptions will be ignored (same behavior as the POSIX rm -f command).

    Changed in version 3.8: The missing_ok parameter was added.
```

**Comment by khaeru** [comment]
Thanks for the suggested fix! Will make a PR to resolve this as #222.

**Comment by khaeru** [comment]
Note that because of:
> Changed in version 3.8: The missing_ok parameter was added.

…we need to check with `Path.exists()` to avoid increasing our minimum Python version.
